prime-router/src/main/kotlin/cli/OktaCommands.kt

```diff
@@ -62,26 +68,53 @@ class LoginCommand : OktaCommand(
         .choice("local", "test", "staging", "prod")
         .default("local", "local")
 
+    private val manualLogin by option(
```

Comment:
This name was a little confusing.  Just to confirm this flag changes whether to use an API key configured as an env var or to log in and get a temporary token?

Reply:
I'm thinking it might make more sense to have the default behavior be the current login page process, and instead have this new method run if specified

Reply:
Updated the name to be clearer, and made the current behavior be the default